python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md

```diff
@@ -85,55 +85,45 @@ done
 
 ## 4. Run Optimized Models (Experimental)
 The examples below show how to run the **_optimized HuggingFace model implementations_** on Intel NPU, including
-- [Llama2-7B](./llama.py)
-- [Llama3-8B](./llama.py)
-- [Llama3.2-1B](./llama.py)
-- [Llama3.2-3B](./llama.py)
+- [Llama2-7B](./llama2.py)
+- [Llama3-8B](./llama3.py)
+- [Llama3.2-1B](./llama3.py)
+- [Llama3.2-3B](./llama3.py)
 - [Qwen2-1.5B](./qwen.py)
 - [Qwen2.5-3B](./qwen.py)
 - [Qwen2.5-7B](./qwen.py)
 - [MiniCPM-1B](./minicpm.py)
 - [MiniCPM-2B](./minicpm.py)
 - [Baichuan2-7B](./baichuan2.py)
 
-### Recommended NPU Driver Version for MTL Users
-#### 32.0.100.2540
-Supported models: Llama2-7B, Llama3-8B, Qwen2-1.5B, MiniCPM-1B, MiniCPM-2B, Baichuan2-7B
-
-### Recommended NPU Driver Version for LNL Users
-#### 32.0.100.2625
-Supported models: Llama2-7B, MiniCPM-1B, Baichuan2-7B
-#### 32.0.101.2715
-Supported models: Llama3-8B, MiniCPM-2B, Qwen2-1.5B, Qwen2.5-7B
-
 ### Run
 ```cmd
 :: to run Llama-2-7b-chat-hf
-python llama.py
+python llama2.py --save-directory <converted_model_path>
```

Comment:
--repo-id-or-model-path ?

Reply:
> --repo-id-or-model-path ?

yeah, have updated the default value in the command.